src/aws/state.tf.dist

```diff
@@ -0,0 +1,9 @@
+# documentation to mange the state in AWS:  https://www.terraform.io/docs/backends/types/s3.html
```

Comment:
should you add `src/aws/state.tf.dist` to the `.gitignore` file?

Reply:
It's a dist file, it's here as an example and should be committed.

Reply:
sorry I meant `src/aws/state.tf` to the .gitignore

Reply:
it is : `src/aws/**/state.tf`